app/components/projects-page-selector.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component, PropTypes } from 'react';
+
+class PageSelector extends Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+    this.renderPageButtons = this.renderPageButtons.bind(this);
+  }
+
+  handleChange(page) {
+    this.props.onChange(page);
+  }
+
+  renderPageButtons(current, total) {
```

Comment:
Does this refer to current page and total pages? If so, I'd rename them to something like that to make it clearer